webpack/ForemanInventoryUpload/Components/ExcludePackagesSwitcher/ExcludePackagesSwitcherActions.js

```diff
@@ -0,0 +1,30 @@
+import API from 'foremanReact/API';
+import { inventoryUrl } from '../../ForemanInventoryHelpers';
+import {
+  EXCLUDE_PACKAGES_TOGGLE,
+  EXCLUDE_PACKAGES_TOGGLE_ERROR,
+} from './ExcludePackagesSwitcherConstants';
+
+export const handleToggle = currentExcludePackages => async dispatch => {
+  const toggledExcludePackages = !currentExcludePackages;
+  try {
+    const {
+      data: { excludePackages },
+    } = await API.post(inventoryUrl('installed_packages_inclusion'), {
+      value: toggledExcludePackages,
+    });
+    dispatch({
+      type: EXCLUDE_PACKAGES_TOGGLE,
+      payload: {
+        excludePackages,
+      },
+    });
+  } catch (error) {
+    dispatch({
+      type: EXCLUDE_PACKAGES_TOGGLE_ERROR,
```

Comment:
Currently there is no code that catches this action. We need one of three choices here:

1. Add reducer code to handle this action, and add a component to handle the state change
2. Publish a toast message instead of a new action
3. Add a generic reducer and a generic component to handle its changes, so all errors could be refactored to emit the same action.

Reply:
I think a toast would be the best fit, and it could become a generic component :+1: